source/unit/Fixture.ooc

```diff
@@ -210,3 +175,39 @@ Test: class {
+expect: func ~double (value: Double, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~ldouble (value: LDouble, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~llong (value: LLong, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~ullong (value: ULLong, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+
+expect: func ~floatvector2d (value: FloatVector2D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~floatvector3d (value: FloatVector3D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~floatpoint2d (value: FloatPoint2D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~floatpoint3d (value: FloatPoint3D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~quaternion (value: Quaternion, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~floateuclidtransform (value: FloatEuclidTransform, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~floatrotation3d (value: FloatRotation3D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~floattransform2d (value: FloatTransform2D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~floattransform3d (value: FloatTransform3D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~intvector2d (value: IntVector2D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~intvector3d (value: IntVector3D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~intpoint2d (value: IntPoint2D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~intpoint3d (value: IntPoint3D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~floatcomplex (value: FloatComplex, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~floatmatrix (value: FloatMatrix, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~inttransform2d (value: IntTransform2D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~floatbox2d (value: FloatBox2D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~intbox2d (value: IntBox2D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~floatshell2d (value: FloatShell2D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
+expect: func ~intshell2d (value: IntShell2D, constraint: Constraint) { Fixture expect(Cell new(value), constraint) }
```

Comment:
`expect: static func ~any <T> (value: T, constraint: Constraint) { This expect(Cell new(value), constraint) }`
Looks like a bug in Rock since the C compiler bailed out.
`error: lvalue required as unary ‘&’ operand`
@fredrikbryntesson Is this a priority?